tsconfig.json

```diff
@@ -17,5 +17,5 @@
     "outDir": "dist"
   },
   "include": ["./src/*", "./src/**/*"],
-  "exclude": ["./src/**/*.test.ts", "./src/**/mocks"]
```

Comment:
Similarly, this was overly strict and turned off type checking on all test files which isn't ideal. It should be just the mocks that produced "hot spots" in compilation (see https://github.com/theopensystemslab/planx-core/pull/165). I'll test this against a pizza on `planx-new`